packages/components/src/Chips/InternalChipDismissible/hooks/useInternalChipDismissible.ts

```diff
@@ -38,7 +38,9 @@ export function useInternalChipDismissible({
   onClick,
   onCustomAdd,
 }: InternalChipDismissibleProps) {
-  const ref = useRef<HTMLDivElement>(null);
```

Comment:
floating UI wants you to use React state for the "attachTo" so it can stay up to date. real refs don't trigger updates, so won't be able to keep up.